tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,312 @@
+---
+title: TLS Connections to Dedicated Tier
+summary: Introduce TLS connection in TiDB Dedicated Tier.
+---
+
+# TLS Connections to Dedicated Tier
+
+On TiDB Cloud, TLS connections are one of the basic security practices for connecting to Dedicated Tier clusters. We support you to configure multiple TLS connections from client, application and development tools to protect your data transmission security. Dedicated Tier cluster only supports TLS 1.2 and TLS 1.3, and does not support TLS 1.0 and TLS 1.1 versions.
+
+For each Dedicated Tier cluster, the server TLS certificate is hosted on [AWS Certificate Manager (ACM)](https://aws.amazon.com/certificate-manager/). ACM Private CA keys are stored securely in AWS-managed hardware security modules (HSMs) that meet [FIPS 140-2 Level 3](https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3139) security standards to protect your TLS private keys when Dedicated Tier requests AWS ACM to create it.
```

Comment:
For "when Dedicated Tier request AWS ACM to create it", what does "it" refer to?

Reply:
updated，Please check  again